Vulnerablity Detector - Fix efficacy tests

## Description
Efficacy tests are failing in the master branch with the following message:
```
FAILED engine/source/vdscanner/qa/test_efficacy_log.py::test_false_negatives[run_process_and_monitor_log21] - AssertionError: The test failed because some expected lines were not found
assert False
 +  where False = all(dict_values([True, False, False, False]))
 +    where dict_values([True, False, False, False]) = <built-in method values of dict object at 0x7fb71f015840>()
 +      where <built-in method values of dict object at 0x7fb71f015840> = {'Match found for OS: windows_server_2008 for vulnerability: CVE-2022-21983 due to default status.': False, 'No match due to default status for OS: windows_server_2008, Version: 6.1.7600 while scanning for Vulnerability: CVE-2022-21981': True, "No remediation available for OS 'windows_server_2008' on Agent '1001' for CVE: 'CVE-2016-0133', discarding.": False, "Remediation for OS 'windows_server_2008' on Agent '1001' has been found. CVE: 'CVE-2022-21983', Remediation: 'KB5034768'.": False}.values
```

The detailed log and workflow where the issue occurred can be found at the following link: [GitHub Actions Workflow](https://github.com/wazuh/wazuh/actions/runs/11576089077/job/32224209277?pr=26380)

## DoD
- [x] Issue identified
- [x] Issue fixed

--- a/src/engine/source/vdscanner/qa/test_false_negative_data/022/Readme.md
+++ b/src/engine/source/vdscanner/qa/test_false_negative_data/022/Readme.md
@@ -11,5 +11,6 @@ Vulnerability detection validation for **_windows_server_2008_** operating syste
 
 | CVE IDs        | Hotfix    | Expected       |
 |----------------|-----------|----------------|
-| CVE-2022-26904 | N/A       | Vulnerable     |
-| CVE-2022-26904 | KB5012649 | Not Vulnerable |
+| CVE-2022-21981 | N/A       | Not Vulnerable |
+| CVE-2022-21983 | N/A       | Not Vulnerable |
+| CVE-2016-0133  | N/A       | Not Vulnerable |

--- a/src/engine/source/vdscanner/qa/test_false_negative_data/022/expected_001.out
+++ b/src/engine/source/vdscanner/qa/test_false_negative_data/022/expected_001.out
@@ -1,6 +1,5 @@
 [
     "No match due to default status for OS: windows_server_2008, Version: 6.1.7600 while scanning for Vulnerability: CVE-2022-21981",
-    "Match found for OS: windows_server_2008 for vulnerability: CVE-2022-21983 due to default status.",
-    "Remediation for OS 'windows_server_2008' on Agent '1001' has been found. CVE: 'CVE-2022-21983', Remediation: 'KB5034768'.",
-    "No remediation available for OS 'windows_server_2008' on Agent '1001' for CVE: 'CVE-2016-0133', discarding."
+    "No match due to default status for OS: windows_server_2008, Version: 6.1.7600 while scanning for Vulnerability: CVE-2022-21983",
+    "No match due to default status for OS: windows_server_2008, Version: 6.1.7600 while scanning for Vulnerability: CVE-2016-0133"
 ]

